shark/iree_utils/_common.py

```diff
@@ -80,10 +80,8 @@ def check_device_drivers(device):
 # Installation info for the missing device drivers.
 def device_driver_info(device):
     if device in ["gpu", "cuda"]:
-        print(
-            "nvidia-smi not found, please install the required drivers from https://www.nvidia.in/Download/index.aspx?lang=en-in"
-        )
+        return "nvidia-smi not found, please install the required drivers from https://www.nvidia.in/Download/index.aspx?lang=en-in"
```

Comment:
BTW I ran black without --check and it did the comments that way

Reply:
Did you set `--line-length 127`?  because it would have failed in the CI